src/pages/products/sgid/transportation/mile-reference-posts.astro

```diff
@@ -0,0 +1,141 @@
+---
+import Layout from '@layouts/DataPage.astro';
+
+import type { IMetadata, IPageMetadata } from '@models/products/sgid/types';
+import { ProductType, SgidCategory } from '@models/products/sgid/types';
+
+import HubDownloads from '@components/data/HubDownloads.astro';
+import Metadata from '@components/data/Metadata.astro';
+
+import CardWithPopularLink from '@components/page/CardWithPopularLink.astro';
+import CardWithSmallLink from '@components/page/CardWithSmallLink.astro';
+import Contacts from '@components/page/Contacts.astro';
+import DefinitionListItem from '@components/page/DefinitionListItem.astro';
+import ExternalLink from '@components/page/ExternalLink.astro';
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import { dataPages } from '@data/downloadMetadata';
+
+export const metadata: IMetadata = {
+  title: 'Utah UDOT Mile Reference Posts',
+  description: `Physical milepost sign locations along Utah highways.`,
+  stewards: ['Utah Department of Transportation (UDOT)'],
+  type: ProductType.POINT,
+  category: SgidCategory.TRANSPORTATION,
+};
+
+const page: IPageMetadata = {
+  ...metadata,
+  application: 'https://maps.udot.utah.gov/wadocuments/Apps/Stationing/',
+  updateHistory: [`2021`],
```

Comment:
**todo**: this needs to be verified and updated

Reply:
waiting to hear back.